ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -49,7 +53,18 @@ def __init__(self, *args, **kwargs):
         self.metadata_collector = MetadataCollector(self._config, self.log)
         self.stats_collector = StatsCollector(self._config, self.send_metrics_from_properties, self.log)
 
+        self.queue_manager_process_pattern = None
+
+        self.check_initializations.append(self.parse_config)
+
     def check(self, _):
+        if not self.check_queue_manager_process():
+            self.log.debug('Process not found, skipping check run')
```

Comment:
```suggestion
            self.log.info('Process not found, skipping check run')
```
If any of the log lines should be info I think it should be this ones and not the ones inside the check_queue_manager_process method